pyramid/url.py

```diff
@@ -34,40 +34,29 @@
 def parse_url_overrides(kw):
     """Parse special arguments passed when generating urls.
 
-    The supplied dictionary is mutated, popping arguments as necessary.
-    Returns a 6-tuple of the format ``(app_url, scheme, host, port,
-    qs, anchor)``.
+    The supplied dictionary is mutated when we pop arguments.
+    Returns a 6-tuple of the format:
+
+      ``(app_url, scheme, host, port, qs, anchor)``.
     """
-    anchor = ''
-    qs = ''
-    app_url = None
-    host = None
-    scheme = None
-    port = None
+    app_url = kw.pop('_app_url', None)
+    scheme = kw.pop('_scheme', None)
+    host = kw.pop('_host', None)
+    port = kw.pop('_port', None)
+    query = kw.pop('_query', '')
+    anchor = kw.pop('_anchor', '')
 
-    if '_query' in kw:
-        query = kw.pop('_query')
+    qs = ''
+    if query:
         if isinstance(query, string_types):
             qs = '?' + url_quote(query, QUERY_SAFE)
-        elif query:
+        else:
             qs = '?' + urlencode(query, doseq=True)
 
-    if '_anchor' in kw:
-        anchor = kw.pop('_anchor')
+    if anchor:
         anchor = url_quote(anchor, ANCHOR_SAFE)
-        anchor = '#' + anchor
-
-    if '_app_url' in kw:
-        app_url = kw.pop('_app_url')
-
-    if '_host' in kw:
-        host = kw.pop('_host')
-
-    if '_scheme' in kw:
-        scheme = kw.pop('_scheme')
-
-    if '_port' in kw:
-        port = kw.pop('_port')
+        if anchor.startswith('#') == False:
```

Comment:
Please fix this PEP8 issue:

https://travis-ci.org/Pylons/pyramid/jobs/229568331#L176